jib-core/src/main/java/com/google/cloud/tools/jib/image/LayerEntry.java

```diff
@@ -56,6 +59,25 @@
   public LayerEntry(Path sourceFile, AbsoluteUnixPath extractionPath) {
     this.sourceFile = sourceFile;
     this.extractionPath = extractionPath;
+    this.permissions = Files.isDirectory(sourceFile) ? 0755 : 0644;
```

Comment:
Let's have these as constants

Reply:
Just wondering if this (i.e., explicitly setting directories to 0755 and files to 0644) changes the current behavior, because it seems like we haven't done anything about permissions.

Reply:
0755 and 0644 were the existing defaults from `TarArchiveEntry`, so it shouldn't have changed anything.